app/javascript/react/screens/App/Overview/OverviewReducer.js

```diff
@@ -227,8 +237,47 @@ export default (state = initialState, action) => {
         .set('isRejectedAllArchivedPlanRequestsWithTasks', true)
         .set('isFetchingAllArchivedPlanRequestsWithTasks', false);
 
+    case `${V2V_FETCH_CLUSTERS}_PENDING`:
+      return state.set('isFetchingClusters', true);
     case `${V2V_FETCH_CLUSTERS}_FULFILLED`:
-      return state.set('clusters', action.payload.data.resources);
+      return state
+        .set('clusters', action.payload.data.resources)
+        .set('isFetchingClusters', false)
+        .set('isRejectedClusters', false)
+        .set('errorClusters', null);
+    case `${V2V_FETCH_CLUSTERS}_REJECTED`:
+      return state
+        .set('errorClusters', action.payload)
+        .set('isRejectedClusters', true)
+        .set('isFetchingClusters', false);
+
```

Comment:
tiny nothing, inconsistent spacing between reducer cases 🔍

Reply:
removed empty lines in that reducer